lib/dotcom_web/templates/customer_support/index.html.eex

```diff
@@ -7,6 +7,7 @@
           <%= fa "check-circle" %>
           Thank you for your feedback.
           <br>Your comments have been sent to our Customer Support team.
+          <br>If you included a contact email, please expect a confirmation email with an incident number within a few minutes. If not, please re-submit your feedback, or call with feedback at 617-222-3200.
```

Comment:
suggestion (non-blocking): use a link to enable easy calling from mobile e.g. `<a href="tel:1-562-867-5309">1-562-867-5309</a>`